src/TestInfrastructure/VsObjectModelMocks/ConfigurationMock.cs

```diff
@@ -38,124 +38,37 @@ public PropertiesMock Properties
 
         #region Configuration
 
-        ConfigurationManager Configuration.Collection
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        ConfigurationManager Configuration.Collection => throw new NotImplementedException();
 
-        string Configuration.ConfigurationName
-        {
-            get
-            {
-                return this.configurationName;
-            }
-        }
+        string Configuration.ConfigurationName => configurationName;
 
-        DTE Configuration.DTE
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        DTE Configuration.DTE => throw new NotImplementedException();
 
-        string Configuration.ExtenderCATID
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        string Configuration.ExtenderCATID => throw new NotImplementedException();
 
-        object Configuration.ExtenderNames
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        object Configuration.ExtenderNames => throw new NotImplementedException();
 
-        bool Configuration.IsBuildable
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        bool Configuration.IsBuildable => throw new NotImplementedException();
 
-        bool Configuration.IsDeployable
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        bool Configuration.IsDeployable => throw new NotImplementedException();
 
-        bool Configuration.IsRunable
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        bool Configuration.IsRunable => throw new NotImplementedException();
 
-        object Configuration.Object
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        object Configuration.Object => throw new NotImplementedException();
 
-        OutputGroups Configuration.OutputGroups
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        OutputGroups Configuration.OutputGroups => throw new NotImplementedException();
 
-        object Configuration.Owner
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        object Configuration.Owner => throw new NotImplementedException();
 
-        string Configuration.PlatformName
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        string Configuration.PlatformName => "x64";
```

Comment:
@duncanp-sonar this is what I really changed. Sorry, I got carried away with the others...